src/module/config.js

```diff
@@ -1400,6 +1400,39 @@ SFRPG.CR_EXP_LEVELS = [
     1638400
 ];
 
+SFRPG.XPTable  = [
```

Comment:
This would be better as an object of objects, with the key as the CR value.
```js
SFRPG.XPTable  = {
  "1/8": {minXP: 0, totalXP: 50, indOneThree: 15, indFourFive: 15, indSixPlus: 10},
  "1/6": {minXP: 50, totalXP: 65, indOneThree: 20, indFourFive: 15, indSixPlus: 10},
}
```
Alternatively for easier key, value lookup, and the ability to store numbers as keys, you could use a Map. Also what does `ind` stand for?

Reply:
ind was standing in for individual, so indOneThree was individual XP award per player when 1-3 players are in the combat. indFourFive and indSixPlus were the same. I can probably update them to something more descriptive like perPlayerXP123, perPlayerXP45, perPlayerXP6?

Reply:
`perPlayerXP` works well